src/Components/Server/test/ElementReferenceJsonConverterTest.cs

```diff
@@ -4,19 +4,30 @@
 using System.IO;
 using System.Text;
 using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.JSInterop;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Components
 {
     public class ElementReferenceJsonConverterTest
     {
-        private readonly ElementReferenceJsonConverter Converter = new ElementReferenceJsonConverter();
+        private readonly IJSRuntime TestRuntime;
+
+        private readonly ElementReferenceJsonConverter Converter;
+
+        public ElementReferenceJsonConverterTest()
+        {
+            TestRuntime = new TestJsRuntime();
```

Comment:
This is fine as-is, but there's a cheaper way to do this especially if you aren't doing anything with the passed in value:

```suggestion
            TestRuntime = Mock.Of<IJSRuntime>();
```